### DISCUSSION TITLE
Brew on linux in Chrome OS container?

### DISCUSSION THREAD

**Discussion by emninger** [Homebrew/discussions#635] [Linux]
Does that work?

**Comment by SMillerDev** [comment]
https://docs.brew.sh/Homebrew-on-Linux if all of the requirements laid out here are met it will likely work.

**Reply by emninger** [reply]
Thanks! By default there is Debian, and do not like sudo apt get (i'm a hardcore pacman fan ;) )

**Reply by iMichka** [reply]
We have a bunch of docker files that might be of interest for you: https://github.com/Linuxbrew/docker
